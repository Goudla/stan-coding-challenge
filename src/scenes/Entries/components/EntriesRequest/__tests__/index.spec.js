import React from 'react';
import TestRenderer from 'react-test-renderer';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import { EntriesRequest } from '../';

describe('<EntriesRequest />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<EntriesRequest loadEntries={() => {}} />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });

  it('should call loadEntries on mount', () => {
    const loadEntries = jest.fn();
    TestRenderer.create(<EntriesRequest loadEntries={loadEntries} />);
    expect(loadEntries).toHaveBeenCalled();
  });
});
