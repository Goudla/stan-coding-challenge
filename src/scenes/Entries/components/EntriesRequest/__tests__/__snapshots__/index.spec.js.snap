// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EntriesRequest /> Should render 1`] = `null`;
